test/test_minitest-display.rb

```diff
@@ -16,7 +16,7 @@ def test_equality
       end
     TESTCASE
 
-    assert_output(/BasicTest/)
+    assert_output(/BasicTest |/)
```

Comment:
Why do we need this changes?

Reply:
This was asserting that it will find the text `BasicTest` in the output, but when all the test cases of that class are passing, we won't be able to see the class name in the output, to puts an `OR` option.

Reply:
https://github.com/bradleymarques/minitest-display/commit/daf2fa7f57b27e6df121d304c09109217ebbb1a6

update minitest 5, this is already been worked on, but we may need to change the source code of this gem, should we?

Reply:
I think we should not change the source code? We should fix the minimal change required.

cc: @ad-ys